guacamole/src/main/java/org/apache/guacamole/rest/APIRequest.java

```diff
@@ -58,6 +68,12 @@ public APIRequest(HttpServletRequest request,
 
         super(request);
 
+        // Grab the remote host info.
+        this.remoteHost = request.getRemoteHost();
+
+	// Grab the remote ip info.
```

Comment:
This poor comment is misaligned.

Also, to match the style of the rest of the guac codebase, one-line non-block comments should not end with a period (see the style of nearby lines). Same goes for the comment just above.